### PR TITLE
fluentd: add configurable syslog drain

### PR DESCRIPTION
**Pull request by ddollar**

### CODE REVIEW

File: docs/installation/production-rack/aws.md [hunk 1]
```diff
@@ -37,10 +37,11 @@ The following environment variables are required:
...
-| `region`    | `us-east-1`   | AWS Region                                       |
\ No newline at end of file
+| Name        | Default       | Description                                                           |
+| ----------- | ------------- | --------------------------------------------------------------------- |
+| `cidr`      | `10.1.0.0/16` | CIDR range for VPC                                                    |
+| `node_disk` | `20`          | Node disk size in GB                                                  |
+| `node_type` | `t3.small`    | Node instance type                                                    |
+| `private`   | `true`        | Put nodes in private subnets behind NAT gateways                      |
+| `region`    | `us-east-1`   | AWS Region                                                            |
+| `syslog`    |               | Forward logs to a syslog endpoint (e.g. `tcp+tls://example.org:1234`) |
```
Comment (beastawakens) [inline docs/installation/production-rack/aws.md:47]:
```suggestion
| `syslog`    |               | Forward all logs to one or more comma delimited syslog endpoints (e.g. `tcp+tls://example.org:1234` or `tcp+tls://example.org:1234,tcp://example.com:5678) |
```

File: docs/installation/production-rack/azure.md [hunk 2]
```diff
@@ -52,7 +52,8 @@ The following environment variables are required:
...
-| Name        | Default          |
-| ----------- | ---------------- |
-| `node_type` | `Standard_D3_v3` |
-| `region`    | `eastus`         |
\ No newline at end of file
+| Name        | Default          | Description                                                           |
+| ----------- | ---------------- | --------------------------------------------------------------------- |
+| `node_type` | `Standard_D3_v3` | Node instance type                                                    |
+| `region`    | `eastus`         | Azure region                                                          |
+| `syslog`    |                  | Forward logs to a syslog endpoint (e.g. `tcp+tls://example.org:1234`) |
```
Comment (beastawakens) [inline docs/installation/production-rack/azure.md:59]:
```suggestion
| `syslog`    |               | Forward all logs to one or more comma delimited syslog endpoints (e.g. `tcp+tls://example.org:1234` or `tcp+tls://example.org:1234,tcp://example.com:5678) |
```

File: docs/installation/production-rack/do.md [hunk 3]
```diff
@@ -38,8 +38,9 @@ Go to https://cloud.digitalocean.com/account/api/tokens and generate a new Space
...
-| `node_type`     | `s-2vcpu-4gb` |
-| `region`        | `nyc3`        |
-| `registry_disk` | `50Gi`        |
\ No newline at end of file
+| Name            | Default       | Description                                                           |
+| --------------- | ------------- | --------------------------------------------------------------------- |
+| `node_type`     | `s-2vcpu-4gb` | Node instance type                                                    |
+| `region`        | `nyc3`        | Digital Ocean region                                                  |
+| `registry_disk` | `50Gi`        | Registry disk size                                                    |
+| `syslog`        |               | Forward logs to a syslog endpoint (e.g. `tcp+tls://example.org:1234`) |
```
Comment (beastawakens) [inline docs/installation/production-rack/do.md:46]:
```suggestion
| `syslog`    |               | Forward all logs to one or more comma delimited syslog endpoints (e.g. `tcp+tls://example.org:1234` or `tcp+tls://example.org:1234,tcp://example.com:5678) |
```

File: docs/installation/production-rack/gcp.md [hunk 4]
```diff
@@ -52,4 +52,5 @@ The following environment variables are required:
 | ------------- | --------------- | ---------------------------------------------------------------------------------------- |
 | `node_type`   | `n1-standard-1` | Node instance type                                                                       |
 | `preemptible` | `true`          | Use [preemptible](https://cloud.google.com/compute/docs/instances/preemptible) instances |
-| `region`      | `us-east1`      | GCP Region                                                                               |
\ No newline at end of file
+| `region`      | `us-east1`      | GCP Region                                                                               |
+| `syslog`      |                 | Forward logs to a syslog endpoint (e.g. `tcp+tls://example.org:1234`)                    |
```
Comment (beastawakens) [inline docs/installation/production-rack/gcp.md:56]:
```suggestion
| `syslog`    |               | Forward all logs to one or more comma delimited syslog endpoints (e.g. `tcp+tls://example.org:1234` or `tcp+tls://example.org:1234,tcp://example.com:5678) |
```

File: docs/installation/production-rack/aws.md (hunk 1 above)
Comment (ddollar) [reply docs/installation/production-rack/aws.md:47]: While this functionality is in there trying to keep it simple for now. Not sure how many people would even use this

Comment (beastawakens) [reply docs/installation/production-rack/aws.md:47]: Fair enough, I guess it would be unusual to have multiple logging providers 😆

**Pull request merged**